scripts/src/main/resources/scripts/devon

```diff
@@ -72,24 +72,61 @@ then
   mkdir -p ~/.devon  
   echo "Copying devon CLI script to your home directory..."
   cp "${BASH_SOURCE:-$0}" ~/.devon/devon
+  cp "${DEVON_IDE_HOME}"/scripts/autocomplete ~/.devon/
+  AUTOCOMPLETION="source ~/.devon/autocomplete"
+  
   if [ ! -e ~/.bashrc ]
   then
     touch ~/.bashrc
   fi
   if ! grep -q "alias devon=" ~/.bashrc
   then
     echo "Installing devon alias to your .bashrc"
-    echo -e '\nalias devon="source ~/.devon/devon"\ndevon' >> ~/.bashrc
+    {
+      echo -e '\nalias devon="source ~/.devon/devon"\ndevon'
+      echo -e "${AUTOCOMPLETION}"
```

Comment:
This approach will not add the autocompletion feature to computers where devonfw-ide has already been "installed" before in an older version.

Reply:
This is correct! Should we change this or we say to the people, please reinstall the ide.